struct.go

```diff
@@ -33,6 +33,8 @@ type BridgeConfig struct {
 	PublicRead bool
 	// UseAsync enables asynchronous operations with go routines.
 	UseAsync bool
+	// Pinata authorization JWT  
```

Comment:
We can modify this comment as discussed above. The goal is to be as descriptive as possible.